lib/mix/tasks/nimble_template.bump_version.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Mix.Tasks.NimbleTemplate.BumpVersion do
+  @shortdoc "Bump the template into specific version."
+
+  @moduledoc """
+  #{@shortdoc}
+
+  - Hex package: https://hex.pm/packages/nimble_template
+  - Github: https://github.com/nimblehq/elixir-templates
+
+  # Usage
+
+  - mix help nimble_template.bump_version # Print help
+  - mix nimble_template.bump_version [new_version] # Bump the template into the [new_version].
```

Comment:
How about:

```suggestion
  - mix nimble_template.bump_version [new_version] # Bump the template version to the [new_version].
```

Reply:
updated in https://github.com/nimblehq/elixir-templates/pull/219/commits/4b0f3f281db7c58f8b52e4e75f6ab469ff79e985